src/pools/D3MAavePool.sol

```diff
@@ -19,9 +19,9 @@ pragma solidity ^0.8.14;
 import "./ID3MPool.sol";
 
 interface TokenLike {
-    function approve(address, uint256) external returns (bool);
+    function balanceOf(address)         external view returns (uint256);
```

Comment:
same as above

Reply:
Ok, for TokenLike let's keep the alignment as before.